### PR TITLE
GDB-8986 Fix svg downloads

### PR DESCRIPTION
**Pull request by yordanalexandrov**
## What
On pages class relationships, class hierarchy and domain range graph, the export svg button yields a broken file

## Why
The selector for the svg export is too broad "svg" and selects all svgs on the screen

## How
- added a parameter to the export function to take a selector and passed appropriate selectors

### CODE REVIEW

File: src/js/lib/common/svg-export.js [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
         });
         return cssStr;
     }
 
-    function generateBase64ImageSource() {
+    function generateBase64ImageSource(svgSelectorClass) {
+        if (!svgSelectorClass) {
+            svgSelectorClass = "svg";
+        }
+        console.log(svgSelectorClass);
```
Comment (svilenvelikov) [inline src/js/lib/common/svg-export.js:19]: logger

**Review by svilenvelikov — DISMISSED** [review]

**Comment by yordanalexandrov** [comment]
Jenkins retry build

**Comment by yordanalexandrov** [comment]
Jenkins retry build

**Review by svilenvelikov — APPROVED** [review]

**Comment by yordanalexandrov** [comment]
Jenkins retry build

**Comment by yordanalexandrov** [comment]
Jenkins retry build

**Pull request merged**